Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "regorus"
+description = "A fast, lightweight Rego (OPA policy language) interpreter"
 version = "0.1.0"
 edition = "2021"
+license-file = "LICENSE"
```

Comment:
Is there a way to include multiple licenses? Say of third-party code?

Reply:
Cargo only supports one license file: https://github.com/rust-lang/cargo/issues/5933 but it does support multiple license expressions like: `license = "MIT OR Apache-2.0"`. I'll try to combine licenses in one license file.

Reply:
We want the project to be MIT licensed; only some borrowed parts to be differently licensed. We can leave the license file as-is till we find out if there is a precedent on how what we want is accomplished.